config/inovelli/lzw60.xml

```diff
@@ -0,0 +1,139 @@
+<Product Revision="1" xmlns="https://github.com/OpenZWave/open-zwave">
```

Comment:
This seems to be missing information for the `113` command class. So motion triggers end up being a value of 'unknown' for both on and off. Rendering this kind of useless for motion sensing

Reply:
I am not having any issues with motion sensing with my setup. Are you using OZW 1.6?  This config was written for 1.6 (because 1.4 is no longer supported),  and I believe there were some significant [changes](https://github.com/OpenZWave/open-zwave/wiki/OpenZWave-1.6-Release-Notes#alarm-notification-commandclass) with the Alarm Command Class in the current version. 

This is how it shows up in my ozwcache_*.xml file:
````
			<CommandClass id="113" name="COMMAND_CLASS_NOTIFICATION">
				<Compatibility />
				<State>
					<CCVersion>8</CCVersion>
					<InNif>true</InNif>
					<StaticRequests>0</StaticRequests>
				</State>
				<Instance index="1" />
				<Value type="list" genre="user" instance="1" index="7" label="Home Security" units="" read_only="true" write_only="false" verify_changes="false" poll_intensity="0" min="0" max="0" vindex="0" size="2">
					<Help>Home Security Alerts</Help>
					<Item label="Clear" value="0" />
					<Item label="Motion Detected at Unknown Location" value="8" />
				</Value>
				<Value type="byte" genre="user" instance="1" index="256" label="Previous Event Cleared" units="" read_only="true" write_only="false" verify_changes="false" poll_intensity="0" min="0" max="255" value="8">
					<Help>Previous Event that was sent</Help>
				</Value>
			</CommandClass>
````

Reply:
I'm using zwave2mqtt with version 1.6.1 of openzwave. My ozwcahce:

```
<CommandClass id="113" name="COMMAND_CLASS_NOTIFICATION">                                                                                                                             
                                <Compatibility />                                                                                                                                                             
                                <State>                                                                                                                                                                     
                                        <CCVersion>3</CCVersion>                                                                                                                                             
                                        <InNif>true</InNif>                                                                                                                                                  
                                        <StaticRequests>0</StaticRequests>                                                                                                                                    
                                </State>                                                                                                                                                                      
                                <Instance index="1" />                                       
                                <Value type="list" genre="user" instance="1" index="8" label="Unknown" units="" read_only="true" write_only="false" verify_changes="false" poll_intensity="0" min="0" max="0"
                                        <Help></Help>                                                                                                                                                         
                                        <Item label="Unknown" value="0" />                                                                                                                                    
                                        <Item label="Unknown" value="2" />                                                                                                                                    
                                        <Item label="Unknown" value="3" />                                                                                                                                   
                                </Value>                                                                                                                                                                     
                                <Value type="int" genre="config" instance="1" index="514" label="Automatically Clear Events" units="ms" read_only="false" write_only="false" verify_changes="false" poll_inten
                                        <Help></Help>                                                                                                                                                         
                                </Value>                                                                                                                                                                      
                        </CommandClass> 
```

Reply:
I am also using zwave2mqtt, with Openzwave 1.6.0 in a docker image.  I just updated to the latest docker image (1.6.1049), and tried refreshing the node into, but it didnt change anything (its still working, and my ozwcache file did not change)

Is your NotificationCCTypes.xml file up-to-date in your config directory?

Its interesting that yours is showing an index = 8.  Based on the NotificationCCTypes.xml file, it should be index = 7 (Home Security)

Reply:
I’ll do some spelunking to see if I can fix it. I’m at least encouraged that it can work. Thanks!